src/dataDisplay/tableHeaderCell.jsx

```diff
@@ -1,32 +1,363 @@
+            this.setState({
+                minWidth: currentWidth,
+            });
+        }
+        this.debouncedUpdateExpandedWidth = _.debounce(
+            (initialLoad) => {
+                const {
+                    shouldResetWhiteSpaceStyle,
+                    setResizableCellsWordWrapping,
+                    stickyTableContainerWidth,
+                    resizableColumnWidthPercentage,
+                    adjacentStickyColumnsTotalWidth,
+                    forceTableUpdate,
+                    stickyColumnCount,
+                } = this.props;
+                const {
+                    widthPercentage: currentWidthPercentage,
+                    minWidth,
+                } = this.state;
+                // eslint-disable-next-line max-len
+                const shouldUpdateExpandedWidth = isResizable && minWidth > 0 && shouldResetWhiteSpaceStyle;
+
+                if (shouldUpdateExpandedWidth) {
+                    // eslint-disable-next-line max-len
+                    const maxWidthPercentages = this.getMaxWidthPercentages(stickyTableContainerWidth);
+                    setResizableCellsWordWrapping(false);
+                    // This ensures the header will readjust according to the content
+                    // after a browser resize event.
+                    this.header.current.style.width = '1%';
+                    this.header.current.style.minWidth = '';
+                    const updatedExpandedWidth = _.get(this, 'header.current.clientWidth', 0) + HEADER_CELL_WIDTH_OFFSET;
+                    // eslint-disable-next-line max-len
+                    const adjustedAdjacentPercentage = (adjacentStickyColumnsTotalWidth / stickyTableContainerWidth) * 100;
+                    const shouldDisableResizeHandler = stickyColumnCount > 1 &&
+                        // eslint-disable-next-line max-len
+                        updatedExpandedWidth + adjacentStickyColumnsTotalWidth > stickyTableContainerWidth * maxWidthPercentages[0];
+                    // eslint-disable-next-line max-len
+                    let expandedWidthPercentage = (updatedExpandedWidth / stickyTableContainerWidth) * 100;
+                    let widthPercentage;
+
+                    if (initialLoad) {
+                        if (resizableColumnWidthPercentage) {
+                            // eslint-disable-next-line max-len
+                            widthPercentage = resizableColumnWidthPercentage - adjustedAdjacentPercentage;
+                        } else {
+                            widthPercentage = ((minWidth / stickyTableContainerWidth) * 100);
+                        }
+                    } else {
+                        widthPercentage = currentWidthPercentage;
+                    }
+
+                    // it never goes further than maximun allowed
+                    const adjustedMaxWIdth = maxWidthPercentages[1] - adjustedAdjacentPercentage;
+                    // eslint-disable-next-line max-len
```

Comment:
Please remove all the `eslint-disable-next-line max-len` comments and fix the error.

Reply:
done!